crates/turbopack-ecmascript/src/chunk/chunk_type.rs

```diff
@@ -11,8 +11,8 @@ use super::{
     EcmascriptChunk, EcmascriptChunkContent, EcmascriptChunkItem, EcmascriptChunkingContext,
 };
 
+#[turbo_tasks::value(shared)]
 #[derive(Default)]
-#[turbo_tasks::value]
 pub struct EcmascriptChunkType {}
```

Comment:
This shouldn't have a public cell method. It should only use `ValueDefault` to make it a singleton